react/components/DropdownProductQuantity.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { FunctionComponent, Fragment } from 'react'
+import { Dropdown } from 'vtex.styleguide'
+import { useCssHandles } from 'vtex.css-handles'
+import { SelectedItem } from 'vtex.product-context'
+
+import { OnChangeCallback, BaseProps } from './BaseProductQuantity'
+
+const MAX_DROPDOWN_VALUE = 10
```

Comment:
How was this value decided?

Reply:
I got it from product-list. I think it would be nice to add a prop so it can be customizable.